README.md

```diff
@@ -16,7 +16,7 @@ To use it in Maven, add the following to your POM:
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>libraries-bom</artifactId>
-      <version>26.0.0</version>
+      <version>26.1.5</version>
```

Comment:
Why did I have to manually update this? Because Release Please didn't update this value https://github.com/googleapis/release-please/issues/1755. Probably the drawback of multi-component setup.